flake.nix

```diff
@@ -0,0 +1,161 @@
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: MIT
+
+{
+  description = "The stablecoin flake";
+
+  nixConfig.flake-registry = "https://gitlab.com/morley-framework/morley-infra/-/raw/main/flake-registry.json";
+
+  inputs.morley-infra.url = "gitlab:morley-framework/morley-infra";
+  inputs.morley.url = "gitlab:morley-framework/morley";
+
+  outputs = { self, flake-utils, haskell-nix, morley-infra, morley, ... }:
+    (flake-utils.lib.eachSystem [ "x86_64-linux" ] (system:
```

Comment:
We used to provide a static binary for releases, but seems like it doesn't make a lot of sense to add such derivation to the flake currently